gslab_scons/builders/build_matlab.py

```diff
@@ -53,10 +54,8 @@ def build_matlab(target, source, env):
     # Matlab options
     if misc.is_unix():
         options = '-nosplash -nodesktop'
-    elif sys.platform == 'win32':
-        options = '-nosplash -minimize -wait'
     else:
-        raise Exception("Unknown OS")
+        options = '-nosplash -minimize -wait'
 
     # Run matlab on source file
     shutil.copy(source_file, 'source.m')
```

Comment:
There is no obvious way to run a script in another directory, so I'd support keeping the current implementation for now.